pkg/tools/cli/loadtest/service/loader_option.go

```diff
@@ -56,9 +57,21 @@ func WithClient(c grpc.Client) Option {
 // WithConcurrency sets load test concurrency.
 func WithConcurrency(c int) Option {
 	return func(l *loader) error {
-		if c > 0 {
-			l.concurrency = c
+		if c <= 0 {
+			return errors.Errorf("concurrency must be natural number")
```

Comment:
this Errorf always returns nil because args of `Errorf` is nil
https://github.com/vdaas/vald/blob/b541c1adbe6ae3db8d7394a652ae603cf148f242/internal/errors/errors.go#L98-L103

so Could you please change from `Errorf` to `New`

Reply:
Thank you for your comments!
I'll fix it.